### PR TITLE
Fix accept invite user

### PR DESCRIPTION
**Pull request by Luiz-FS**
<p><b>Feature/Bug description:</b> Fix the accept invitation page because it was allowing the user to click the accept invitation button several times which caused a bug in the user profiles that were repeated.</p>

<p><b>Solution:</b> I prevented the invitation from being accepted several times on the server side and hid the button to accept the invitation in the frontend when it has already been clicked, instead, I added a circular progress.</p>

screenshot from 2018-01-25 09-28-03
screenshot from 2018-01-25 09-30-00

<p><b>TODO/FIXME:</b> n/a</p>

### CODE REVIEW

File: backend/handlers/invite_handler.py [hunk 1]
```diff
@@ -70,6 +71,11 @@ def patch(self, user, invite_key):
         data = self.request.body
 
         invite = ndb.Key(urlsafe=invite_key).get()
+
+        Utils._assert(invite.status == 'accepted',
```
Comment (mayzabeel) [inline backend/handlers/invite_handler.py]: SE certificar se esse é o handler para aceitar convite quando convida um membro e quando convida uma instituição. Acredito que outros Handerls mudam o status de um invite de sent to accepted.

Comment (Luiz-FS) [reply backend/handlers/invite_handler.py]: Done

File: backend/handlers/institution_handler.py [hunk 2]
```diff
@@ -134,7 +134,16 @@ def post(self, user, institution_key, inviteKey):
 
         institution = ndb.Key(urlsafe=institution_key).get()
 
-        institution.createInstitutionWithStub(user, inviteKey, institution)
+        invite = ndb.Key(urlsafe=inviteKey).get()
+
+        Utils._assert(invite.status == 'accepted', 
+            "Invitation already accepted",
```
Comment (raonismaneoto) [inline backend/handlers/institution_handler.py]: This invitation has already been accepted is better, isn't it?

File: backend/handlers/invite_handler.py [hunk 3]
```diff
@@ -70,6 +71,11 @@ def patch(self, user, invite_key):
         data = self.request.body
 
         invite = ndb.Key(urlsafe=invite_key).get()
+
+        Utils._assert(invite.status == 'accepted', 
+            "Invitation already accepted",
```
Comment (raonismaneoto) [inline backend/handlers/invite_handler.py]: same thing

File: frontend/invites/new_invite_page.html [hunk 4]
```diff
@@ -2,76 +2,81 @@
...
-                    <div class="md-list-item-text" layout="column">
-                      <h3><b>{{ newInviteCtrl.institution.name }}</b></h3>
-                      <p>{{ newInviteCtrl.institution.email }}</p>
-                      <p>Convidado por : {{ newInviteCtrl.invite.admin_name }}</p>
-                    </div>
-                  </md-list-item>
-                </md-list>
-                <div layout="column" layout-align="center">
-                  <p class="small-text increase-distance md-title green-text">{{newInviteCtrl.isInviteUser() ? 'PREENCHA O FORMULÁRIO PARA SE TORNAR UM MEMBRO:' : 'PREENCHA O FORMULÁRIO PARA CADASTRAR A INSTITUIÇÃO:'}}</p>
+        <form name="accetpInvite" ng-submit="newInviteCtrl.acceptInvite($event)">
```
Comment (andreldsa) [inline frontend/invites/new_invite_page.html]: This form tag should be inside md-card-contetn

**Pull request merged**